dimod/higherorder.py

```diff
@@ -103,8 +104,9 @@ def make_quadratic(poly, strength, vartype=None, bqm=None):
 
     Examples:
 
+        >>> import dimod
```

Comment:
We're moving away from importing for the main package in examples. If you were importing from numpy say you would have an import, but `import dimod` is kind of redundant.